pandas/core/frame.py

```diff
@@ -2568,6 +2568,21 @@ def _ixs(self, i: int, axis: int = 0):
 
             return result
 
+    def _ixs_values(self, i: int) -> Union[np.ndarray, ExtensionArray]:
```

Comment:
let's name this _iget_values for consistency no? (ixs sounds like cross-section, e.g. rows which this is not)

Reply:
"Union[np.ndarray, ExtensionArray]" should be "ArrayLike"

Reply:
@jbrockmendel do you have a preference for the name?